content/en/docs/contribute/style/style-guide.md

```diff
@@ -631,6 +631,34 @@ You can remove ... | You can easily remove ...
 These steps ... | These simple steps ...
 {{< /table >}}
 
+### EditorConfig
+For maintaing a constant coding style we would need you to maintain a  .editorconfig file 
+so that the developers using different editors (like vim , emacs , atom , vscode etc) for the same project can contribute with the same style.
+
+the .editorconfig used here is 
+```
+   [*]
+   end_of_line = lf
+   insert_final_newline = false
+   charset = utf-8
+   max_line_length = 80
+   trim_trailing_whitespace = true
+
+   [*.md]
+   trim_trailing_whitespace = false
+
+   [*.{css,html,js,json,sass,md,mmark,toml,yaml}]
+   indent_style = space
+   indent_size = 2
+
+   [*.{sh}]
+   indent_style = space
+   indent_size = 4
+
+   [Makefile]
+   indent_style = tab
```

Comment:
This is likely to go stale; maybe we can hyperlink to the file instead?

Reply:
@sftm i have added the changes , can you look into it?